src/Features/Core/Portable/CodeRefactorings/MoveType/AbstractMoveTypeService.MoveTypeEditor.cs

```diff
@@ -290,16 +340,6 @@ private void AddPartialModifiersToTypeChain(
                     documentEditor.RemovePrimaryConstructor(node);
                 }
             }
-
-            documentEditor.ReplaceNode(State.TypeNode,
```

Comment:
this was moved to a helper method.  it is still called in both the callers that called this method.  just at different places now.